src/config.ts

```diff
@@ -5,6 +5,7 @@ export interface Config {
     numberOfReposPerCall: number;
     inputFilePath: string;
     outputFilePath: string;
+    daysAgoCreated: number | undefined;
```

Comment:
This could be:
daysAgoCreated?: number;

Reply:
Changing it. Thanks.